src/main/java/org/folio/des/repository/criteria/JsonbNodeConverter.java

```diff
@@ -13,12 +13,12 @@
 
 import org.z3950.zing.cql.CQLTermNode;
 
-public class JsonbNodeToPredicateConverter {
+public class JsonbNodeConverter {
 
   public static final String JSONB_EXTRACT_PATH_TEXT_FUNC = "jsonb_extract_path_text";
 
-  public Predicate convert(CQLTermNode node, CriteriaBuilder cb, Root<?> root) {
-    List<String> fieldNames = getFieldNames(node.getIndex());
+  public static Expression<String> convertToExpression(Root<?> root, String jsonPath, CriteriaBuilder cb){
```

Comment:
You already inject instance of JsonbNodeConverter to CQL2JPACriteria, so these methods can be not static that is preferable

Reply:
Ok, I have fixed